clients/js/test/initClientWithAuth.ts

```diff
@@ -3,31 +3,37 @@ import { CloudClient } from "../src/CloudClient";
 
 const PORT = process.env.PORT || "8000";
 const URL = "http://localhost:" + PORT;
-export const chromaBasic = new ChromaClient({
-  path: URL,
-  auth: { provider: "basic", credentials: "admin:admin" },
-});
-export const chromaTokenDefault = new ChromaClient({
-  path: URL,
-  auth: { provider: "token", credentials: "test-token" },
-});
-export const chromaTokenBearer = new ChromaClient({
-  path: URL,
-  auth: {
-    provider: "token",
-    credentials: "test-token",
-  },
-});
-export const chromaTokenXToken = new ChromaClient({
-  path: URL,
-  auth: {
-    provider: "token",
-    credentials: "test-token",
-    tokenHeaderType: "X_CHROMA_TOKEN",
-  },
-});
-export const cloudClient = new CloudClient({
-  apiKey: "test-token",
-  cloudPort: PORT,
-  cloudHost: "http://localhost",
-});
+export const chromaBasic = () =>
+  new ChromaClient({
+    path: URL,
+    auth: { provider: "basic", credentials: "admin:admin" },
+  });
+export const chromaTokenDefault = () =>
+  new ChromaClient({
+    path: URL,
+    auth: { provider: "token", credentials: "test-token" },
+  });
+export const chromaTokenBearer = () =>
+  new ChromaClient({
+    path: URL,
+    auth: {
+      provider: "token",
+      credentials: "test-token",
+      tokenHeaderType: "AUTHORIZATION",
+    },
+  });
```

Comment:
It looks like the token header type was dropped off of this config in the previous PR but let me know if this configuration is no longer necessary:

https://github.com/chroma-core/chroma/pull/1970/files#diff-a6a9add9fb7cdb9cd98f93430b5c049f2352f497ac1213141f05dc3cc473cb3bL19